test/cli/test_commands.py

```diff
@@ -798,6 +799,39 @@ def test_pre_run_hook_module_not_found(testdir, cli):
     assert lines[9] == "Aborted!"
 
 
+def test_add_case(testdir, cli, schema_url):
+    module = testdir.make_importable_pyfile(
+        hook="""
+            import schemathesis
+            import click
+
+            @schemathesis.hooks.register
+            def add_case(case):
+                if not case.headers:
+                    case.headers = dict()
```

Comment:
As for small stylistic preference and small performance win I prefer having dicts as literals if it is possible - `{}`